conf/plants/allowed_species.json

```diff
@@ -0,0 +1,111 @@
+[
+    "actinidia_chinensis",
+    "aegilops_tauschii",
+    "amborella_trichopoda",
+    "ananas_comosus",
+    "arabidopsis_halleri",
+    "arabidopsis_lyrata",
+    "arabidopsis_thaliana",
+    "arabis_alpina",
+    "beta_vulgaris",
+    "brachypodium_distachyon",
+    "brassica_napus",
+    "brassica_oleracea",
+    "brassica_rapa",
+    "camelina_sativa",
+    "camelina_sativa",
+    "camelina_sativa",
+    "camelina_sativa",
+    "cannabis_sativa_female",
+    "capsicum_annuum",
+    "chara_braunii",
+    "chlamydomonas_reinhardtii",
+    "chondrus_crispus",
+    "citrullus_lanatus",
+    "citrus_clementina",
+    "coffea_canephora",
+    "corchorus_capsularis",
+    "cucumis_melo",
+    "cucumis_sativus",
+    "cyanidioschyzon_merolae",
+    "cynara_cardunculus",
+    "daucus_carota",
+    "dioscorea_rotundata",
+    "eragrostis_curvula",
+    "eragrostis_tef",
+    "galdieria_sulphuraria",
+    "glycine_max",
+    "gossypium_raimondii",
+    "helianthus_annuus",
+    "hordeum_vulgare",
+    "hordeum_vulgare_goldenpromise",
+    "ipomoea_triloba",
+    "leersia_perrieri",
+    "lupinus_angustifolius",
+    "malus_domestica_golden",
+    "manihot_esculenta",
+    "marchantia_polymorpha",
+    "medicago_truncatula",
+    "musa_acuminata",
+    "nicotiana_attenuata",
+    "nymphaea_colorata",
+    "olea_europaea_sylvestris",
+    "oryza_barthii",
+    "oryza_brachyantha",
+    "oryza_glaberrima",
+    "oryza_glumipatula",
+    "oryza_indica",
+    "oryza_longistaminata",
+    "oryza_meridionalis",
+    "oryza_nivara",
+    "oryza_punctata",
+    "oryza_rufipogon",
+    "oryza_sativa",
+    "ostreococcus_lucimarinus",
+    "panicum_hallii_fil2",
+    "panicum_hallii_hal2",
+    "phaseolus_vulgaris",
+    "physcomitrella_patens",
+    "pistacia_vera",
+    "populus_trichocarpa",
+    "prunus_avium",
+    "prunus_dulcis",
+    "prunus_persica",
+    "rosa_chinensis",
+    "saccharum_spontaneum",
+    "saccharum_spontaneum",
+    "saccharum_spontaneum",
+    "saccharum_spontaneum",
+    "saccharum_spontaneum",
+    "selaginella_moellendorffii",
+    "setaria_italica",
+    "setaria_viridis",
+    "solanum_lycopersicum",
+    "solanum_tuberosum",
+    "sorghum_bicolor",
+    "theobroma_cacao_criollo",
+    "theobroma_cacao_matina",
+    "trifolium_pratense",
+    "triticum_aestivum",
+    "triticum_aestivum",
+    "triticum_aestivum",
+    "triticum_aestivum",
+    "triticum_aestivum",
+    "triticum_aestivum_cadenza",
+    "triticum_aestivum_claire",
+    "triticum_aestivum_paragon",
+    "triticum_aestivum_robigus",
+    "triticum_aestivum_weebil",
```

Comment:
Your suggestion makes sense, as it would take less time to copy some dnafrags around, etc. but not sure if our pipelines (mainly `PrepareMasterDatabaseForRelease`) are prepared for this scenario, so removing them from master would only make them to be copied again next release?

Reply:
PrepareMaster can remove a genome, yes, and the rest would work.
Maybe you need to code something, though: all the species reported by the metadata service that are not in `allowed_species.json` should be moved to the "removed_species" categories, to make sure the GenomeDBs are retired.